server/src/main/java/org/elasticsearch/index/query/TermsQueryBuilder.java

```diff
@@ -500,4 +399,255 @@ protected QueryBuilder doRewrite(QueryRewriteContext queryRewriteContext) {
 
         return this;
     }
+
+    public static abstract class Values extends AbstractCollection implements Writeable {
```

Comment:
`TermsQueryBuilder` provides a public method `getValues()`, which returns a `Values`. It seems that `Values` is not used outside now because the `getValues()` is used in this pattern: 
```
return new TermsQueryBuilder(rewrittenFieldName, terms.getValues());
```
So maybe this class should be public since there is such a method? :)